internal/gui/gui.go

```diff
@@ -64,18 +64,18 @@ func StartGUI(papermcAPI paper_api.PapermcAPI) {
 		case *tcell.EventResize:
 			s.Sync()
 		case *tcell.EventKey:
-			if ev.Key() == tcell.KeyCtrlC {
+			if ev.Key() == tcell.KeyCtrlC || string(ev.Rune()) == "q" {
 				return
-			} else if ev.Key() == tcell.KeyEscape {
+			} else if ev.Key() == tcell.KeyEscape || string(ev.Rune()) == "h" {
```

Comment:
Im not convinced about having "h" as an escape key, as this is also not a common VI keybind